source/metadata.py

```diff
@@ -11,10 +11,18 @@
 # This is used by automation like the Pinecil updater
 
 
-if len(sys.argv) != 2:
+if len(sys.argv) < 2:
     print("Requires the output json name as an arg")
     exit(1)
 
+# Explicitly provide target file to scan for json output
+ModelName = None
+if len(sys.argv) == 3 and sys.argv[2] is not None and sys.argv[2] != "":
```

Comment:
What is the latter half of this test for (for arg 2); can you actually have those fail?

Reply:
Just thought the more checks the better but here it seems too much indeed. Sorry, I use/read/write python few-liners sporadically so I always open to learn zen of python on practice from more proficient people.